x-pack/plugins/security_solution/cypress/tasks/risk_scores/saved_objects.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RiskScoreEntity } from '../../screens/entity_analytics';
+import { getRiskScoreTagName } from '../../screens/entity_analytics';
+import { RISK_SCORE_SAVED_OBJECTS_URL, SAVED_OBJECTS_URL } from '../../urls/risk_score';
+
+export const deleteSavedObjects = (
```

Comment:
Please, consider moving all the API calls to the `api_calls` folder inside the tasks one.